### DISCUSSION TITLE
v5.8.0: no changelog/release but container published

### DISCUSSION THREAD

**Discussion by muhlba91** [tsightler/ring-mqtt#953] [Support Q&A] [answered]
Hi,

using Renovate, I have seen that a new container was published using the `v5.8.0` tag.
However, looking at the releases and tags in this repository, there's no mentioning of a `v5.8.0`.

1/ Is this an official release?
2/ Is there a changelog for it?

Thank you! 😃

**Comment by tsightler** [comment, marked as answer]
Changelog is always in the same place:

https://github.com/tsightler/ring-mqtt/blob/main/docs/CHANGELOG.md

I've decided to stop doing "releases" in Github, there's little reason to download the tar files so it's mostly just waste.
